lib/helpers/projection/isSubpath.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+/*!
+ * Determines if `path2` is a subpath of or equal to `path1`
+ *
+ * @param {string} path1
+ * @param {string} path2
+ * @return {Boolean}
+ */
+
+module.exports = function isSubpath(path1, path2) {
+  const path1Parts = path1.split('.');
```

Comment:
I'd much rather this function be `path1 === path2 || path1.startsWith(path2 + '.') || path2.startsWith(path1 + '.')`. The `split()` function is relatively expensive performance-wise, and we try to avoid it where possible.

Reply:
Good idea! Fixed a minor bug in your logic and incorporated that suggestion.